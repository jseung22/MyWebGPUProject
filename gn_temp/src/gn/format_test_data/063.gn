source_set("test") {
  a = "a"
  b = "b"
  c = {
    d = "d"
    e = "e"
  }
  f = [
    "f0",
    "f1",
    "f2",
  ]
  deps = [
    "//a",
    "//a/a",
    "//a/b",
    "//a:a",
    "//a:b",
    "//b",
    c.e,
    c.d,
    ":a",
    ":b",
    f[2],
    f[0],
    "a",
    "a/a",
    "a/b",
    "a:a",
    "a:b",
    "b",
    a,
    b,
    c.d,
    f[0],
    f[1],
  ]

  public_deps = []
  if (condition) {
    public_deps += [
      "//a",
      "//a/a",
      "//a:a",
      ":a",
      "a",
      "a/a",
      "a:a",
      a,
    ]

    deps += [
      ":d",
      ":e",
      # Some comment attached to an item. Should be moved with the item
      # when it is sorted.
      ":a",
      ":b",
      # Comment attached to a duplicated item. Should be concatenated to
      # the comment from the other item.
      ":a",
    ]
  }

  # Sort lists with "deps" suffix as "deps".
  foo_deps = [
    "//a",
    ":z",
  ]

  # Likewise for visibility
  visibility = [
    "//b:*",
    "//a",
    "//b/*",
    ":z",
  ]
}
