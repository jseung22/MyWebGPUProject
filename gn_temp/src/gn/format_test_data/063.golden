source_set("test") {
  a = "a"
  b = "b"
  c = {
    d = "d"
    e = "e"
  }
  f = [
    "f0",
    "f1",
    "f2",
  ]
  deps = [
    ":a",
    ":b",
    "a",
    "a:b",
    "a/a",
    "a/b",
    "b",
    "//a",
    "//a:b",
    "//a/a",
    "//a/b",
    "//b",
    a,
    b,
    c.d,
    c.e,
    f[0],
    f[1],
    f[2],
  ]

  public_deps = []
  if (condition) {
    public_deps += [
      ":a",
      "a",
      "a/a",
      "//a",
      "//a/a",
      a,
    ]

    deps += [
      # Some comment attached to an item. Should be moved with the item
      # when it is sorted.
      # Comment attached to a duplicated item. Should be concatenated to
      # the comment from the other item.
      ":a",
      ":b",
      ":d",
      ":e",
    ]
  }

  # Sort lists with "deps" suffix as "deps".
  foo_deps = [
    ":z",
    "//a",
  ]

  # Likewise for visibility
  visibility = [
    ":z",
    "//a",
    "//b:*",
    "//b/*",
  ]
}
